src/libraries/System.Text.Encodings.Web/ref/System.Text.Encodings.Web.csproj

```diff
@@ -11,12 +11,14 @@
     <Compile Include="System.Text.Encodings.Web.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Memory\ref\System.Memory.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime\ref\System.Runtime.csproj" />
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.Extensions\ref\System.Runtime.Extensions.csproj" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' or
-                         $(TargetFramework.StartsWith('net4'))">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+    <Reference Include="System.Resources.ResourceManager" />
```

Comment:
Is this needed? The public surface (reference assembly) does not seem to dependent on ResourceManager.

What are the changes in this file trying to achieve?

Reply:
I had retargeted to netcoreapp31 (partly because 21 will be out of support by the time this ships, 30 is already out of support, partly because 31 introduced new APIs I'm using), and this was reacting to the fact that types got moved around leading up to the 31 timeframe.